modules/solid_mechanics/src/materials/ReturnMappingModel.C

```diff
@@ -26,6 +26,8 @@ validParams<ReturnMappingModel>()
       "relative_tolerance", 1e-5, "Relative convergence tolerance for sub-newtion iteration");
   params.addParam<Real>(
       "absolute_tolerance", 1e-20, "Absolute convergence tolerance for sub-newtion iteration");
+  params.addParam<Real>(
+      "max_inelastic_incr", 1e-4, "The maximum inelastic strain increment allowed in a time step");
```

Comment:
I really don't like this name.  If we spared a few characters and made this `max_inelastic_increment`, it would be more consistent with our other parameters.